iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties

```diff
@@ -227,45 +247,54 @@ dn_seed_config_node=127.0.0.1:10710
 # The maximum session idle time. unit: ms
 # Idle sessions are the ones that performs neither query or non-query operations for a period of time
 # Set to 0 to disable session timeout
+# effectiveMode: first_start
 # Datatype: int
 # dn_session_timeout_threshold=0
 
+# weather enable thrift compression
+# effectiveMode: restart
 # Datatype: boolean
 # dn_rpc_thrift_compression_enable=false
 
-# if true, a snappy based compression method will be called before sending data by the network
-# Datatype: boolean
-# this feature is under development, set this as false before it is done.
-# dn_rpc_advanced_compression_enable=false
```

Comment:
dn_rpc_advanced_compression_enable is used
<img width="913" alt="image" src="https://github.com/apache/iotdb/assets/32640567/00d8c3d9-b296-4a3e-bcf0-e852f3fff410">